### DISCUSSION TITLE
Rainbow armor on armorstands

### DISCUSSION THREAD

**Discussion by titivermeesch** [Zrips/CMI#4595] [Feature Requests]
**Description of feature/improvement request:**

Rainbow armor does not seem to work or armorstands, not sure if this is a bug or if it's a missing feature. But it would be nice to have it available to use

**Comment by EkonC** [comment]
Nice idea, maybe in the inventory as well (if it's possible) and on mobs. This could be probably  heavy on the side of servers, so maybe an option to enable or disable it.
